dashboard/app/views/admin_reports/hoc_signups.html.haml

```diff
@@ -1,39 +1,70 @@
 - content_for :head do
   %script{src:'https://www.google.com/jsapi'}
 
+:css
+  .center {
+    text-align: center;
+  }
+  .right {
+    text-align: right;
+  }
+
 %h1
-  Hour of Code 2015 Signups
+  Hour of Code Signups
 
 .by_day
-  #chart_by_day
+  %table
+    %tr
+      %td
+        #signups_by_day
+      %td
+        #cumulatives_by_day
   %table
     %tr
       %th Day
-      %th Count
-    - signups_by_day.each do |day|
+      %th HOC 2014 Count
+      %th HOC 2015 Count
+    - data_by_day.each do |day|
       %tr
-        %td
+        %td.center
           %span= day[0]
-        %td
-          %span= day[1]
+        %td.right
+          %span= day[1][0]
+        %td.right
+          %span= day[1][1]
 
 - content_for :body_scripts do
   :javascript
-    // Load the Visualization API and the appropriate packages, setting a callback
-    // to run when the API is loaded.
-    google.load('visualization', '1.0', {'packages':['corechart']});
-    google.setOnLoadCallback(drawChart);
-    var signups_by_day = #{signups_by_day.try(:to_json)}
-    // The callback that creates and populates the data table, instantiates the
-    // chart, and draws it.
-    function drawChart() {
-      var data = new google.visualization.DataTable();
-      data.addColumn('string', 'Date');
-      data.addColumn('number', 'Count');
-      data.addRows(signups_by_day);
+    var data_by_day = #{data_by_day.try(:to_json)};
+
+    if (data_by_day.length > 0) {
+      // Generate the signups and cumulative breakdowns from the data, if data exists.
+      signups_by_day = [[data_by_day[0][0], data_by_day[0][1][0], data_by_day[0][1][1]]];
+      cumulatives_by_day = [[data_by_day[0][0], data_by_day[0][1][0], data_by_day[0][1][1]]];
+      for (var i = 1; i < data_by_day.length; i++) {
```

Comment:
So that we can reference cumulatives[prev_day] when computing cumulatives[next_day].